extensions/bad-merchant-ux/shopify.extension.toml

```diff
@@ -27,11 +27,6 @@ default_placement = "ORDER_INDEX"
 # https://shopify.dev/docs/api/customer-account-ui-extensions/unstable/configuration#api-access
 api_access = true
 
-# Gives your extension access to make external network calls, using the
-# JavaScript `fetch()` API. Learn more:
-# https://shopify.dev/docs/api/customer-account-ui-extensions/unstable/configuration#network-access
-network_access = true
```

Comment:
drive by change this is not needed and blocks deployments if not granted in the partner dashboard. seems to be unnecessary friction